wpeverest/user-registration
# UR-574 Fix -Tooltip is-removed when clicked multiple times on field-in builder

### All Submissions:
PR dependency in https://github.com/wpeverest/user-registration-pro/pull/339






### Changes proposed in this Pull Request:



Previously if tooltip enabled first and required enabled later order was ?* and when tooltip enabled clicking multiple times on active field on form builder would disable tooltip. This PR resolve this issue

### How to test the changes in this Pull Request:

1.While making form first enable tooltip than enable required than *? should be shown instead of ?*
2.After enabling tooltip in a filed click that field multiple times in form builder tooltip shouldnot be disabled

### Types of changes:




### Other information:




### Changelog entry

> Fix -Tooltip removed when clicked multiple times on field in builder.
